README.md

```diff
@@ -80,29 +80,29 @@ fn main() {
 }
 ```
 
-Tests use a two-level indentation syntax: the outer most level of indentation
-defines a command name (multiple command names can be specified, as in the
-above); each command name can then define tests for one or more of `status:
-<success|failure|<int>>` (where `success` and `failure` map to platform
-specific notions of a command completing successfully or unsuccessfully
-respectively and `<int>` is a signed integer checking for a specific exit code,
-on platforms that support it), `stderr: [<string>]`, `stdout: [<string>]`.
-
-In essence, each keyword under a command name is a test for that command. The
-above file contains 4 tests: the `Compiler` should succeed (e.g. return a `0`
-exit code when run on Unix), and its `stderr` output should warn about an
-unused variable on line 12; and the resulting binary should succeed and produce
-`Hello world` on `stdout`.
-
-Lines not mentioned are not tested: for example, the above file does not state
-whether the `Compiler`s `stdout` should have content or not (but note that the
-line `stdout:` on its own would state that the `Compiler` should have no
-content at all). `stderr`/`stdout` tests can use `...` as a simple wildcard: if
-a line consists solely of `...`, it means "match zero or more lines"; if a line
-begins with `...`, it means "match the remainder of the line only"; if a line
-ends with `...`, it means "match the start of the line only". A line may start
-and end with `...`. `stderr`/`stdout` matches ignore leading/trailing
-whitespace and newlines, but are case sensitive.
+Test data is specified with a two-level indentation syntax: the outer most
+level of indentation defines a test command (multiple command names can be
+specified, as in the above); each test command can then define sub-tests for
+one or more of `status: <success|failure|<int>>` (where `success` and `failure`
```

Comment:
I wonder if we should bullet list this part for readability and to break up the text a bit?